src/components/RemoveDialog.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/macro';
+import { ButtonPrimary } from './Button';
+import { ButtonSecondary } from './Button';
+
+export default function RemoveDialog({
+  issueId,
+  setShowRemoveDialog,
+  handleRemoveIssue,
+}) {
+  return (
+    <Wrapper>
+      <ButtonPrimary
+        children={'CANCEL'}
+        onClick={() => {
+          setShowRemoveDialog(false);
+        }}
+      />
+      <ButtonSecondary
+        children={'REMOVE'}
+        onClick={() => {
+          handleRemoveIssue(issueId);
+        }}
+      />
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: inherit;
+  flex-wrap: wrap;
```

Comment:
If you use `flex-wrap`, why do you bet with `display: inherit` that the parent element has `display:flex`? What happens if you change the parent's display property? Is there a reason why you not just put `flex`here as well?

Reply:
There's no specific reason, I changed it to flex